obspy/imaging/waveform.py

```diff
@@ -37,6 +40,10 @@
 from obspy.core.util import (FlinnEngdahl, createEmptyDataChunk,
                              locations2degrees)
 from obspy.core.util.decorator import deprecated_keywords
+from obspy.imaging.util import ObsPyAutoDateFormatter
+
+
+MINMAX_ZOOMLEVEL_WARNING_TEXT = "Warning: Zooming into MinMax Plot!"
```

Comment:
No, I'm not against the warning. I just mean it should indicate why zooming can't or shouldn't be done a bit more clearly, even if it's just to say something as simple as it's unsupported or something.